drivers/soundwire/stream.c

```diff
@@ -700,8 +700,11 @@ static int sdw_bank_switch(struct sdw_bus *bus, int m_rt_count)
 	else
 		addr = SDW_SCP_FRAMECTRL_B0;
 
-	sdw_fill_msg(wr_msg, NULL, addr, 1, SDW_BROADCAST_DEV_NUM,
-		     SDW_MSG_FLAG_WRITE, wbuf);
+	ret = sdw_fill_msg(wr_msg, NULL, addr, 1, SDW_BROADCAST_DEV_NUM,
+			   SDW_MSG_FLAG_WRITE, wbuf);
+	if (ret < 0)
+		return ret;
```

Comment:
will ret > 0 ? no ?  why not if (!ret) ?

Reply:
> will ret > 0 ? no ? why not if (ret) ?

ret will never > 0. However, use `if (ret < 0)` can keep consistent.

Reply:
I don't think any error can be returned here given the parameters given.

This is a false positive IMHO.

Reply:
> I don't think any error can be returned here given the parameters given.
> 
> This is a false positive IMHO.

Indeed, let me close the PR.